assets/javascripts/openqa.js

```diff
@@ -626,15 +626,21 @@ function renderHttpUrlAsLink(value) {
     value = String(value);
   }
   const urlRegex = /https?:\/\/[^\s,]*/g;
+  const arrayOfUrlsRegex = /^(https?:\/\/[^\s,]*)(\s+https?:\/\/[^\s,]*)+$/g;
   let lastIndex = 0;
   let match;
+  let arrayOfUrls = arrayOfUrlsRegex.exec(value) !== null;
   while ((match = urlRegex.exec(value)) !== null) {
     if (match.index > lastIndex) {
       fragment.appendChild(document.createTextNode(value.substring(lastIndex, match.index)));
     }
     const a = document.createElement('a');
     a.href = a.textContent = match[0];
     fragment.appendChild(a);
+    if (oneUrlPerLine && arrayOfUrls) {
```

Comment:
small thing but I guess `oneUrlPerLine` should be checked once `arrayOfUrls` is true. Not the other way around.